### ISSUE TITLE
email_hash is not on the README

### ISSUE THREAD

**Issue by osvik** [greenpeace/gpi-techlab-pixelcounter#1]
The email_hash parameter is not in the README file. Is it still supported?

**Comment by tottaz** [comment]
yes, it is still supported, I will add it to the README.md - I will re-do the whole RREADME>md in the next few days.
